src/lib/utils/parse-user-agent.ts

```diff
@@ -0,0 +1,15 @@
+export function parseUserAgent(userAgent: string) {
+  if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(userAgent)) {
+    return 'send-to-email';
```

Comment:
Could we change to 'mobile', it looks a bit confusing when we do the validation `operatingSystem === "sent-to-email"`